go.mod

```diff
@@ -3,22 +3,18 @@ module github.com/jpmorganchase/quorum-account-plugin-hashicorp-vault
 go 1.13
 
 require (
+	github.com/consensys/goquorum-crypto-secp256k1 v0.0.1
+	github.com/ethereum/go-ethereum v1.10.8 // indirect
```

Comment:
this is an issue - we need to make sure that neither go-ethereum nor goquorum are in the dependency list otherwise we have to use lgpl license.  can you check if `github.com/consensys/goquorum-crypto-secp256k1` is puling that dependency in.